logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/SendCallingMessage.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.call
+
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.message.MessageContent
+import java.util.UUID
+
+suspend fun CallManager.senCallingMessage(conversationId: ConversationId, data: String) {
+    val messageContent =  MessageContent.Calling(data)
+    val message =  Message(UUID.randomUUID().toString(), messageContent, conversationId)
```

Comment:
As we are creating/sending a new message to other recipients we are creating a new `UUID`?

Reply:
yes, it's a setup message that will be passed to AVS lib on the device B